src/Microsoft.ML.Data/Evaluators/Metrics/ClusteringMetrics.cs

```diff
@@ -26,30 +26,30 @@ public sealed class ClusteringMetrics
         /// Note however, that this metric will only decrease if the number of clusters is increased,
         /// and in the extreme case (where each distinct example is its own cluster) it will be equal to zero.
         /// </summary>
-        public double AvgMinScore { get; }
+        public double AverageMinimumScore { get; }
```

Comment:
>AverageMinimumScore [](start = 22, length = 19)

From documentation it's unclear what minimum means.

Reply:
AverageDistance perhaps?

---
In reply to: [259518165](https://github.com/dotnet/machinelearning/pull/2700#discussion_r259518165) [](ancestors = 259518165)

Reply:
Mathematical things need mathematical documentation. Could we write some equations here?

Reply:
@wschin https://blog.mindorks.com/good-code-vs-bad-code-263f71e867c1